blog/src/app/blog/items.ts

```diff
@@ -14,6 +15,16 @@ export interface BlogItem {
 }
 
 const items: BlogItem[] = [
+    {
+        title: 'Do World Champions Lose Their Minds? | Dojo Talks',
+        subtitle: 'Jesse, Kostya, and David • May 17, 2024',
+        body: `On today's episode of Dojo Talks, the ChessDojo podcast, GM Jesse Kraai, IM Kostya Kavutskiy, and IM David Pruess talk about the impact of the FIDE World Chess Championship circuit on mental health and the sanity of those who endure it.`,
+        href: '/blog/dojo-talks-lose-mind',
+        image: {
+            src: talksLoseMind,
+            alt: 'Thumbnail',
```

Comment:
Alt should either be a useful description of the image or an empty string if screen readers should ignore the image completely